docs/3.0/develop/settings-and-profiles.mdx

```diff
@@ -1,31 +1,112 @@
 ---
 title: Configure settings and profiles
-description: Prefect settings let you customize your workflow environment, including your Cloud and self-hosted server preferences.
+description: Prefect settings let you customize behavior across different environments.
 ---
 
-Many of Prefect's features—especially interactions with the API—require configuration. You can configure settings through the following methods:
+## Why use settings?
 
-- **Environment Variables**: All Prefect settings can be set using environment variables prefixed with `PREFECT_`. Environment variables are useful for temporarily overriding settings or configuring the runtime environment of a workflow. They take precedence over profile settings, making them ideal for adjustments that should only apply to a single session or process. These can also be set in a `.env` file, which will be automatically applied when using `prefect` in that directory.
+Settings in Prefect help you control how your workflows behave. They let you easily customize Prefect to work the way you need it to, whether you're testing locally or running in production.
 
-- **Profiles**: Prefect profiles store sets of settings locally on your machine. When you activate a profile, its settings are applied, allowing you to switch easily between different configurations. For example, you might use one profile for a self-hosted Prefect server and another for Prefect Cloud.
+Specifically, settings enable:
 
-To view all available settings and their active values from the command line, run:
+- **Environment-Specific Configuration**: Use different settings for development (like detailed logging), testing (like test databases), and production (like your production server) without changing your workflow code.
+
+- **Runtime Flexibility**: Quickly adjust things like retry attempts or logging levels without having to modify and redeploy your workflows.
+
+## Getting Started with Settings
+
+The simplest way declare settings is by creating a `prefect.toml` file in your project directory. For example:
+
+```toml prefect.toml
+# Set more detailed logging while developing
+logging.level = "DEBUG"
+```
+
+Most editors have plugins for TOML that provide syntax highlighting, linting, and autocomplete for `prefect.toml` files. If you use VSCode, we recommend the [Even Better TOML extension](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml).
+
+## Settings sources
+
+You can configure settings via the following sources:
```

Comment:
I think it would be useful if you explicitly state here that these will be listed in order of precedence, or reverse order of precedence in this case. (And keep the list below too, I just think the redundancy is useful in this case)

Reply:
Makes sense! I added an aside that this list is from highest precedence to lowest.

Reply:
Makes sense! I added an aside that this list is from highest precedence to lowest.